doc/classes/EditorSettings.xml

```diff
@@ -420,6 +420,9 @@
 		<member name="editors/3d/selection_box_color" type="Color" setter="" getter="">
 			The color to use for the selection box that surrounds selected nodes in the 3D editor viewport. The color's alpha channel influences the selection box's opacity.
 		</member>
+		<member name="editors/3d/active_selection_box_color" type="Color" setter="" getter="">
+			The color to use for the active selection box that surrounds selected nodes in the 3D editor viewport. The color's alpha channel influences the selection box's opacity.
+		</member>
```

Comment:
```suggestion
		<member name="editors/3d/active_selection_box_color" type="Color" setter="" getter="">
			The color to use for the active selection box that surrounds selected nodes in the 3D editor viewport. The color's alpha channel influences the selection box's opacity.
			[b]Note:[/b] The “active” means that it is one of the selected objects and is the only one object that can be used as a basis for some operations.
		</member>
```
Since active is a new term in Godot, it would be better to have an explanation of it.